content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -182,44 +213,93 @@ Run the following commands on the virtual machine you want to add to the Istio m
     $ sudo apt -y upgrade
     {{< /text >}}
 
+1. Install the root cert to the `/var/run/secrets/istio`
+    
+    {{< text bash >}}
+        $ sudo mkdir -p /var/run/secrets/istio
+        $ sudo cp ~/root-cert.pem /var/run/secrets/istio/root-cert.pem
+    {{< /text >}}
+    
+1. Install the token to the `/var/run/secrets/tokens`
+    {{< text bash >}}
+        $ sudo  mkdir -p /var/run/secrets/tokens
+        $ sudo cp ~/istio-token /var/run/secrets/tokens/istio-token
+    {{< /text >}}
+    
 1. Install the `deb` package containing the Istio virtual machine integration runtime.
 
     {{< text bash >}}
     $ curl -LO https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb
     $ sudo dpkg -i istio-sidecar.deb
     {{< /text >}}
 
-1. Install `root-cert.pem`, `key.pem` and `cert-chain.pem` within the directory `/etc/certs/`.
+1. Install `cluster.env` within `/var/lib/istio/envoy/`.
 
     {{< text bash >}}
-    $ sudo mkdir -p /etc/certs
-    $ sudo cp {root-cert.pem,cert-chain.pem,key.pem} /etc/certs
+    $ sudo cp ~/cluster.env /var/lib/istio/envoy/cluster.env
     {{< /text >}}
 
-1. Install `cluster.env` within `/var/lib/istio/envoy/`.
+1. Add the istiod host to `/etc/hosts`.
 
     {{< text bash >}}
-    $ sudo cp cluster.env /var/lib/istio/envoy
+    $ sudo cp ~/hosts /etc/hosts
     {{< /text >}}
 
-1. Add the istiod host to `/etc/hosts`.
-
+1. Install the root cert to the `/etc/certs`
+    
     {{< text bash >}}
-    $ sudo sh -c 'cat hosts-addendum >> /etc/hosts'
+        $ sudo cp ~/root-cert.pem /var/run/secrets/istio/root-cert.pem
     {{< /text >}}
 
 1. Transfer ownership of the files in `/etc/certs/` and `/var/lib/istio/envoy/` to the Istio proxy.
 
     {{< text bash >}}
-    $ sudo chown -R istio-proxy /etc/certs /var/lib/istio/envoy
+    $ sudo mkdir -p /etc/istio/proxy
+    $ sudo chown -R istio-proxy /var/lib/istio /etc/certs /etc/istio/proxy  /var/run/secrets
+    {{< /text >}}
+
+1. Set the Environment variable for cert rotation cert received and rotated in `/var/run/secrets/istio` 
+    {{< text bash >}}
```

Comment:
no need for export in this block. please remove them.

Reply:
actually, now that i see the next command, the exports may be needed. You may consider changing line 261 to "Export the environment variable for cert rotation. The cert i srotated in `/var/run/secretes/istio`.

Reply:
We should not be doing any of this - must be in sidecar.env. That is why L272 is broken as well

Reply:
create a new sidecar.env to enable the feature